### ISSUE TITLE
VoiceOver doesn't read question titles because `aria-label` is missing from question title HTML elements

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#6859]
T14457 - Re: Accessibility Issue: Aria label missing.
https://surveyjs.answerdesk.io/internal/ticket/details/T14457

related issues:

- https://github.com/surveyjs/survey-library/issues/4121
- https://github.com/surveyjs/survey-library/issues/3421
- https://github.com/surveyjs/survey-library/pull/6210

**Comment by dmitrykurmanov** [comment]
https://stackoverflow.com/questions/42477371/voiceover-not-reading-aria-label-for-links-in-ios-10
https://bugs.webkit.org/show_bug.cgi?id=162578

**Comment by JaneSjs** [comment]
Upon further research, it appears that the title of a Radiogroup question is not being read by a Voice Over screen reader when the Radiogroup appears on the first survey page and the [survey.focusFirstQuestionAutomatic](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model#focusFirstQuestionAutomatic) option is enabled (which is the default setting).

However, the structure of the resulting form page seems to comply with accessibility standards. Additionally, we have tested the survey with other screen readers and confirmed that the title of a radiogroup question is correctly read by them (for example, we tested it with NVDA). Therefore, it seems to be a specific issue that is only reproduced on the Voice Over reader. Unfortunately, we may not be able to address this issue at the moment.

To resolve this issue, we recommend disabling the [survey.focusFirstQuestionAutomatic](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model#focusFirstQuestionAutomatic) option. With this option disabled, Voice Over should correctly read the title of a radiogroup question even if the question appears on the first survey page.
